modules/elements-color-picker/assets/js/editor/commands/apply.js

```diff
@@ -0,0 +1,48 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class Apply extends CommandBase {
+	/**
+	 * Execute the color apply command.
+	 *
+	 * @param value The new color to apply.
+	 * @param trigger The element which triggered the Apply command. Used to show `Selected` text & listen to `mouseleave`.
+	 */
+	apply( { value, trigger } ) {
```

Comment:
```suggestion
	apply( { value, trigger } ) {
```
If the arg is requirement please validate it

Reply:
It's not required.